backend/conf/logger.js

```diff
@@ -1,2 +1,39 @@
-const logger = require('simple-node-logger').createSimpleLogger('project.log');
-module.exports = logger;
\ No newline at end of file
+const simpleNodeLogger = require('simple-node-logger');
+
+const eventTypes = {
+  request: '[REQUEST]',
+  server: '[SERVER]',
+  app: '[APP]'
+}
+
+const logger = simpleNodeLogger.createSimpleLogger({ logFilePath: 'project.log', timestampFormat: 'YYYY-MM-DD - HH:mm:ss.SSS |' });
+
+logger.serverInfo = (message) => {
+  logger.info(`${eventTypes.server} ${message}`);
+};
```

Comment:
Podemos dejarlos así bastante más inlineados.
```suggestion
logger.serverInfo = (message) => logger.info(`${eventTypes.server} ${message}`);
```